manifests/s/StudioRack/StudioRack/0.0.32/StudioRack.StudioRack.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: StudioRack.StudioRack
+PackageVersion: 0.0.32
+Installers:
+- Architecture: x86

```

Comment:
```suggestion
- Architecture: x64
```
![image](https://user-images.githubusercontent.com/12611259/166505941-59256521-3c8c-4d18-8d6f-792ad3b8360e.png)

Reply:
How does this happen? I only updated the link with wingetcreate?